packages/byods/src/token-manager/index.ts

```diff
@@ -21,14 +28,22 @@ export default class TokenManager {
    * @example
    * const tokenManager = new TokenManager('your-client-id', 'your-client-secret');
    */
-  constructor(clientId: string, clientSecret: string, baseUrl: string = PRODUCTION_BASE_URL) {
+  constructor(
+    clientId: string,
+    clientSecret: string,
+    config?: BYODSConfig,
```

Comment:
Same here, would it be better to pass the level directly rather than the whole config object